Chaap/Chaap/Presentation/Main/Segments/CalendarSegmentView.swift

```diff
@@ -8,11 +8,238 @@
 import SwiftUI
 
 struct CalendarSegmentView: View {
+    @StateObject private var viewModel = CalendarSegmentViewModel()
+
     var body: some View {
-        Text("캘린더 세그먼트 뷰입니다.")
+        ZStack {
+            /// 전체화면 배경
+            Color.gray.opacity(0.2)
+                .ignoresSafeArea(.all)
+
+            VStack(spacing: 0) {
+                Spacer()
+
+                VStack(alignment: .center, spacing: 16) {
+                    monthHeader
+
+                    VStack(alignment: .leading, spacing: 5) {
+                        weekdayHeader
+                        calendarGrid
+                    }
+                    .padding(.horizontal, 16)
+                    .padding(.vertical, 0)
+                    .frame(maxWidth: .infinity, alignment: .topLeading)
+
+                    eventsList
+                }
+                .padding(0)
+                .frame(width: 360, alignment: .top)
```

Comment:
frame 지정하지 않고 horizontal padding값으로 크기 조절하면 됩니다!
```suggestion
                .padding(.horizontal, 16)
```